nexus/db-queries/src/db/datastore/deployment.rs

```diff
@@ -892,6 +893,8 @@ impl DataStore {
 
         Ok(Blueprint {
             id: blueprint_id,
+            // TODO these need to be serialized to the database.
```

Comment:
Do we want an issue filed for this?

Reply:
Mentioned in f29f33faec9d40b40911521931bec34d8cd01028.